src/build.props

```diff
@@ -9,9 +9,15 @@
     <Company Condition=" '$(Company)' == '' ">Microsoft</Company>
     <Product Condition=" '$(Product)' == '' ">Microsoft SARIF SDK</Product>
     <Copyright Condition=" '$(Copyright)' == '' ">© Microsoft Corporation. All rights reserved.</Copyright>
+    
+    <!-- VersionPrefix denotes the stable Version of SARIF Specifications. -->
```

Comment:
>denotes the stable Version of SARIF Specification [](start = 23, length = 49)

No. VersionPrefix denotes the assembly version that we will version out of band with the SARIF specification version.